instructure/tinymce-a11y-checker
# Enhance support for Internet Explorer 11

This makes the development "demo" environment work properly in
IE11.  This is probably only useful for testing stuff out in that
environment.

It also fixes an issue that only cropped up in IE11 where a forEach
was being called on a NodeList improperly.

closes CORE-1972

Test Plan:
  - yarn && yarn start
  - Open the up the demo environment in IE11
  - Run the a11y checker, it should work.
